README.md

```diff
@@ -1,4 +1,161 @@
-# Python boilerplate for GitHub tasks
+# Lockdown cafe
 
 - Read [the guideline](https://github.com/mate-academy/py-task-guideline/blob/main/README.md) before start
-- Implement the task described [here](app/main.py)
```

Comment:
Add here please link to `datetime` info: https://www.geeksforgeeks.org/python-datetime-module/

Reply:
Done